### ISSUE TITLE
multiline-if-else: False negatives for android code style

### ISSUE THREAD

**Issue by BraisGabin** [pinterest/ktlint#828]
## Expected Behavior
Extracted from [Android Kotlin style guide](https://developer.android.com/kotlin/style-guide#braces)
> Braces are not required for when branches and if statement bodies which have no else if/else branches and which fit on a single line.
> Braces are otherwise required for any if, for, when branch, do, and while statements, even when the body is empty or contains only a single statement.

I can't stand the same for Kotlin coding conventions (but I think that it should apply too). More about this in #812

## Observed Behavior

This code is not flagged and it should:

```kotlin
if (true) 50
  .toString()
```

```kotlin
if (true) 50
else 55
```

I fixed this issues in a similar rule in detekt https://github.com/detekt/detekt/pull/2835 and https://github.com/detekt/detekt/pull/2051

## Your Environment
* Version of ktlint used: 0.37.2
